javascript/javascript1/week3/distanceTime.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <script>
+
+const travelInformation = {
+  speed: 50,
+  destinationDistance: 432,
+};
+
+
+function travelPeriod(travelInformation){
+    return travelInformation.destinationDistance/travelInformation.speed
+}
+
+function caltravelTime() {
+        time = travelInformation.destinationDistance / travelInformation.speed;
+        let rhours = Math.floor(time) * 60;
+        minutes = (time % 1) * 100;
+        rminutes = Math.floor(minutes);
+        return rhours + rminutes;
+      }
+
+      function timeConvert(caltravelTime) {
+        var num = caltravelTime;
```

Comment:
NEVER use var! I cannot emphasize this enough 😄

Reply:
I have changed them to let,.i will push new file, Thanks